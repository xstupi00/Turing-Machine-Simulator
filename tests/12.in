S a B b
B b C a
B b C b
B b C c
C a D e
C a D f
C a D g
C b E e
C b E f
C b E g
C c I a
C c I b
C c I b
D e G e
D f G f
D g G g
G f B f
B f I f
I f J R
J x F x
ax