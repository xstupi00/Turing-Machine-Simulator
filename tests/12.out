Sax
Bbx
Cax
Dfx
Gfx
Bfx
Ifx
fJx
fFx
